SCloudRandomizer/SearchParams.h

```diff
@@ -7,11 +7,15 @@
 //
 
 #import <Foundation/Foundation.h>
+#import "GlobalData.h"
```

Comment:
oh I think you meant to add it to SearchParamsVC.h? Because its being used in SearchParamsVC.m